scripts/export/export.go

```diff
@@ -0,0 +1,91 @@
+package export
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+
+	app "github.com/cosmos/cosmos-sdk/cmd/gaia/app"
+	"github.com/cosmos/cosmos-sdk/codec"
+	tmtypes "github.com/tendermint/tendermint/types"
+)
+
+// GenesisFile defines the Gaia genesis format
+type GenesisFile struct {
+	GenesisTime     string                   `json:"genesis_time"`
+	ChainID         string                   `json:"chain_id"`
+	ConsensusParams *tmtypes.ConsensusParams `json:"consensus_params"`
```

Comment:
Yeahhhh, so I tested this by running the flow against a local testnet and there is a problem. The problem is as follows:

The network started with `v0.33.0` which uses TM `v0.31.0-dev0` which has `ConsensusParams` that are NOT compatible with `v0.33.2+`. So when you call `tmtypes.GenesisDocFromFile`, it will panic because the `ConsensusParams` params are of a different structure. So we need a way to _read_ the exported genesis file through `v0.31.0-dev0` `ConsensusParams` and _write_ the ported genesis file through `v0.31.3` `ConsensusParams`.

Reply:
I agree, this does not require to be written in Go

Reply:
... that's unfortunate. Is there any way we can pin multiple versions of Tendermint under different names with Go modules and import the old code version?

Reply:
Let's see if @fedekunze can get it to work with modules. If not, we may have to use a non-typed language.